q2cli/builtin/dev.py

```diff
@@ -69,56 +69,71 @@ def refresh_cache():
      "bright_magenta, bright_cyan, or bright_white.")
 
 
-@dev.command(name='install-theme',
+@dev.command(name='import-theme',
              short_help='Install new command line theme.',
-             help=install_theme_help,
+             help=import_theme_help,
              cls=ToolCommand)
 @click.option('--theme', required=True,
               type=click.Path(exists=True, file_okay=True,
                               dir_okay=False, readable=True),
               help='Path to file containing new theme info')
-def install_theme(theme):
+def import_theme(theme):
     import os
     import shutil
     import q2cli.util
-    from q2cli.core.config import CONFIG
-
-    CONFIG.parse_file(theme)
+    from q2cli.core.config import ParserError, CONFIG
+
+    try:
+        CONFIG.parse_file(theme)
+    except ParserError as e:
```

Comment:
One guy that wasn't caught with this:
```pytb
configparser.MissingSectionHeaderError: File contains no section headers.
file: 'example.theme', line: 1
'asd\n'
```

I passed a file that wasn't even plausibly a config.

Reply:
I was only trying to catch the errors I raised not the errors the actual parser raised. Should I just catch all exceptions in that except?

Reply:
Certainly not all, but I think parse errors from configparser fall under the same, expected problems which don't need a traceback category.

Reply:
I don't know if they have a base class for all of them, but they might, which would make catching them easy.

Reply:
Yep, `configparser.Error`, I got rid of my `ParserError` and am raising that instead, so we only need to except that error type